examples/helloworld/greeter_client/main.go

```diff
@@ -47,9 +49,19 @@ const (
 	defaultName = "world"
 )
 
+var withInsecureTLS = flag.Bool("insecure_tls", false, "Use Insecure TLS; suitable for hitting the greeter_server in -use_http mode")
```

Comment:
Because I only implemented ServeHTTP for TLS. That's all Go's http2 currently does.